README.adoc

```diff
@@ -215,6 +215,15 @@ list of available Rake tasks via:
 $ bundle exec rake -T
 ---------------------
 
+=== Code style
+
+The code style is enforced by http://batsov.com/rubocop[RuboCop]. To verify that your code
+against the rules, run:
```

Comment:
To verify that your code follow the [Ruby Style Guid](https://github.com/bbatsov/ruby-style-guide), run the test suite as: